setup/connector_elasticsearch/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
```

Comment:
you must pin the version of elasticsearch
https://github.com/OCA/search-engine/blob/12.0/setup/connector_elasticsearch/setup.py#L6

Reply:
Done